app/models/mission.rb

```diff
@@ -1,3 +1,74 @@
 class Mission < ActiveRecord::Base
-  # Remember to create a migration!
+  has_many :rest_points
+  has_many :passengers
+  has_many :dispatches
+  has_many :trailer_dispatches
+
+  def self.create_new(params)
```

Comment:
this method is way too long! even in our Sinatra applications, we want to follow the OO principles we covered in Phase One. it is incredibly important to keep our methods single responsibility so that they are DRY and easy to read and follow. currently, i'm not even sure i understand what's going on in this method